spec/system/single_item_metadata_spec.rb

```diff
@@ -24,11 +24,12 @@
   it "has expected metadata" do
     visit '/catalog/78348'
     expect(page).to have_content "Midplane neutral density profiles in the National Spherical Torus Experiment"
-
+    find('#show-more-metadata-link').click
```

Comment:
There are two buttons of the page with the text "Show More" so I had to target the button by `id` with `find` before clicking on it.